pkg/cmd/gist/edit/edit.go

```diff
@@ -222,3 +297,36 @@ func updateGist(apiClient *api.Client, hostname string, gist *shared.Gist) error
 
 	return nil
 }
+
+func processFiles(filename string) (string, bool, error) {
+	fileExists := false
+
+	if fi, err := os.Stat(filename); err != nil {
+	} else {
```

Comment:
this conditional is a no-op. should this be an early return? I'm not following why, if this failed, the later glob would help.

Reply:
Yes you're right. I guess I over thought this. The above one too.